guide/designing-products/common-user-flows.md

```diff
@@ -251,13 +251,19 @@ See the [onboarding section](/guide/onboarding/introduction/) to learn more on t
    layout = "float-left"
 %}
 
-It is essential to distinguish between wallet software and the wallet representing the user's "account." This particular experience may be almost invisible or highly elaborate. While some mobile wallet applications choose to create and secure a wallet during the first launch automatically, other wallets provide extensive configuration wizards.
+By _wallet_, we normally mean the wallet [_application_](/guide/glossary/wallet/#wallet-application). Here we will talk about creating the [_Bitcoin wallet_](/guide/glossary/wallet/#bitcoin-wallet), the private key(s) that the software interacts with to control funds on the Bitcoin blockchain.
 
 </div>
 
+A good starting point today is an [HD wallet](/guide/glossary/wallet/#hd-wallet) implemented according to [BIP](/guide/glossary/#bip---bitcoin-improvement-proposal)32, 39, 43, 44, 49 and 84. But not all Bitcoin wallets are created this way, as new standards and technologies have emerged over time.
```

Comment:
What are the new standards? There is no mention of descriptors in this section!

Reply:
The BIP's listed are what I meant by new standards. Are output descriptors captured in any standard that relates to _HD wallets_, or is something that all modern wallet _applications_ should support?

If you were to link to an official spec/description of them would this be it, or something else? https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md

Reply:
@Bosch-0 How do output descriptors impact the user experience? What are the significant differences to the user when a wallet does support or does not support this feature? Perhaps this question is outside the scope of this PR discussion, but I'm looking at it through that lens in relation to the Guide

Reply:
I added BIP 380 (output descriptors) to the list, and removed the confusing last sentence.